doc/backends.md

```diff
@@ -47,3 +47,49 @@ Otherwise there is no configuration required. The test will configure libvirt on
 own. By default, the openSUSE test distribution uses the domain name `openQA-SUT-1`.
 So you can for instance use `virsh dumpxml openQA-SUT-1` to investigate the configuration
 of the running virtual machine.
+
+
+## vagrant
```

Comment:
```suggestion
## Vagrant
```
Same counts for the other occurrences; as it is a name it should start with a capital letter. (Unless it is the literal command of course.)